packages/types/src/methods/sign-psbt.ts

```diff
@@ -3,11 +3,9 @@ import { DefineRpcMethod, RpcRequest, RpcResponse } from '../rpc';
 export type SignPsbtRequest = RpcRequest<'signPstb'>;
 
 export type SignPsbtResponse = RpcResponse<{
-  psbtHex: string;
-  inputToSign: {
-    sigHash: any[];
-  }[];
-  network: any;
+  account: number;
+  hex: string;
+  signAtIndex: number | number[];
```

Comment:
optional type,

what about sighash and public key?

Reply:
Forgot `allowedSighash`, thanks. I think `publicKey` came from the Connect implementation of PSBT signing. I was confused in our code why it was a param? The app didn't explicitly pass it to us? I don't see that it is ever used during signing?

Reply:
I am also questioning `network` in our implementation, where is it ever used?